.gitignore

```diff
@@ -41,4 +43,7 @@ go.work
 go.work.sum
 
 # gitguardian
-.cache_ggshield
\ No newline at end of file
+.cache_ggshield
+# ignore test binary
+bindings/go/plugin/manager/registries/componentversionrepository/testdata/test-plugin
```

Comment:
I think you can just build this into tmp: https://github.com/open-component-model/open-component-model/pull/53/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R40

Reply:
done